packages/core/content-type-builder/admin/src/components/FormModal/forms/index.js

```diff
@@ -68,17 +68,16 @@ const forms = {
         }
 
         if (injectedInputs) {
-          const injectedInputsSection = injectedInputs.map(injectedInput => {
-            if (has(injectedInput, 'sectionTitle')) {
-              // When it has a sectionTitle (including null), use it
-              return injectedInput;
-            }
-
-            // Otherwise add an empty sectionTitle so it is added as a new section
-            return { sectionTitle: null, items: [injectedInput] };
-          });
+          // TODO: Discuss how to handle settings from other plugins
```

Comment:
You can take a look at how i18n injects a section into a field and reuse some of this logic/api

Reply:
Happy to discuss on Discord you know where to find me!

Reply:
Sounds good, messaged you on discord 👍